### PR TITLE
Style option to change tab bar height

### PR DESCRIPTION
**Pull request by Avarel**
image

New option to change the tab bar's height. Using the default option (24.0) should yield no visual difference. The tab add button looks a little off with this change though, so that's up for further design/discussion I think.

### CODE REVIEW

**Comment by Adanos020** [comment]
Yeah, before we merge this, we need to address the add button. Perhaps make its size the same as the close button's?

**Comment by Avarel** [comment]
Done, not a fan though because to make it the same size I basically have to query the font for a size (since the close button has the same size as the galley's height)

**Review by Adanos020 — CHANGES_REQUESTED** [review]
I noticed that the click area of the add button still resizes with the tab bar height: notice how the add button gets highlighted when you hover over the area around it with increased bar height. This needs to be fixed before merging.

**Comment by Avarel** [comment]
Sorry took me a while to get to this, but the click area for the add button has been fixed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # egui_dock changelog
 
+## Unreleased
+- `Style` now includes an option to change the tab's height (url pending).
```
Comment (Adanos020) [inline CHANGELOG.md:4]: Add an `###Added` section above this point

**Review by Adanos020 — CHANGES_REQUESTED** [review]

**Review by Adanos020 — APPROVED** [review]

**Pull request merged**